t/proxy.t

```diff
@@ -1,166 +1,187 @@
-use strict;
+use v5.20.0;
 use warnings;
 
 use Log::Dispatchouli;
 use Test::More 0.88;
 
-my $logger = Log::Dispatchouli->new_tester({
-  ident => 't/proxy.t',
-});
+{
+  my $logger = Log::Dispatchouli->new_tester({
+    ident => 't/proxy.t',
+  });
 
-sub are_events {
-  my ($comment, $want) = @_;
+  my sub are_events {
```

Comment:
CI is failing because my subs are experimental on 5.20 it looks like

Reply:
👍🏽 Fixed.  I just added the CI to main _after_ writing this PR, then rebased. :)